### PR TITLE
CES-9802 - Configure NICs' PXE boot settings for UEFI

### PR DESCRIPTION
**Pull request by digambar15**

### CODE REVIEW

File: src/pilot/config_idrac.py [hunk 1]
```diff
@@ -110,10 +110,51 @@ def configure_nics_boot_settings(
...
+               "PxeDev3EnDis": "Disabled",
+               "PxeDev4EnDis": "Disabled",
+               "PxeDev1Interface": pxe_nic_id,
+               "PxeDev1Protocol": "IPv4",
+               "PxeDev1VlanEnDis": "Disabled"
+               }
+
+            provisioning_mac = nic.mac_address.lower()
+            response = drac_client.set_bios_settings(settings)
+            reboot_required = response['commit_required']
```
Comment (cdearborn) [inline src/pilot/config_idrac.py]: The above line uses the 'commit_required' attribute, which has been deprecated.  It should use the 'is_reboot_required' attribute instead.  See: https://github.com/openstack/python-dracclient/blob/master/dracclient/utils.py#L156

File: src/pilot/config_idrac.py [hunk 2]
```diff
@@ -110,10 +110,51 @@ def configure_nics_boot_settings(
...
+
+            if response['commit_required']:
```
Comment (cdearborn) [inline src/pilot/config_idrac.py]: The above line uses the 'commit_required' attribute, which has been deprecated.  It should use the 'is_commit_required' attribute instead.

File: src/pilot/config_idrac.py (hunk 1 above)
Comment (cdearborn) [reply src/pilot/config_idrac.py]: After discussing the above with Rick, given that this is being developed on a 10.2 cluster, I'm ok with leaving the above as-is for now.

File: src/pilot/config_idrac.py (hunk 2 above)
Comment (cdearborn) [reply src/pilot/config_idrac.py]: After discussing the above with Rick, given that this is being developed on a 10.2 cluster, I'm ok with leaving the above as-is for now.

**Comment by digambar15** [comment]
will fix those attributes in future release

**Pull request merged**